Add ability to add custom bundle (#1083)
acorn-io/acorn: docs/docs/30-installation/02-options.md

## Endpoint domain names
Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [networking page](/running/networking#dns).

## Custom CA bundle

> Acorn allows user to provide custom ca bundle so that users can add own private CA and have acorn trust that CA. The most common use case is for acorn to trust an internal image registry that is signed by private CA.

Reviewer: nit:
```suggestion
Acorn allows a user to provide a custom certificate authority (CA) bundle so that users can add their own private CA that acorn will trust. The most common use case is for acorn to trust an internal image registry that is signed by a private CA.
```

> To do so, you will need to add the following steps:

Reviewer: nit:
```suggestion
To do so, you will need to go through the following steps:
```

1. Provide your CA certificate chain in the following secret.

```bash
kubectl -n acorn-image-system create secret generic cabundle --from-file=ca-certificates.crt=/path/to/your/ca-certificates.crt

kubectl -n acorn-system create secret generic cabundle --from-file=ca-certificates.crt=/path/to/your/ca-certificates.crt
```

:::info

> You must provide **full** CA certificate chain as it will override existing CA certificates in acorn control plane.

Reviewer: ```suggestion
You must provide the **full** CA certificate chain as it will override existing CA certificates in acorn control plane.
```

:::

2. Install acorn with the following option

```bash

> acorn install --use-custom-cabundle

Reviewer: Just a thought around UX, should I be able to pass the name of the secret in the form of `namespace/secret`? This should allow clusters with existing CA secrets to not need to create new ones.

I don't want to block this PR with this, so I'll just put it out their for us to think about. If its something we want to do, we can do it in a follow-up outside of the release.

Author: Good point! @ibuildthecloud brought this up and we think at this point we might not want to make thing more complex.

See https://acorn-io.slack.com/archives/C03FU91U4SX/p1674491780902799?thread_ts=1674490264.688439&cid=C03FU91U4SX

Reviewer: Awesome! Glad to see we already had this thought.
